kong/plugins/log-serializers/basic.lua

```diff
@@ -21,7 +21,11 @@ function _M.serialize(ngx)
   local request_tls
   local request_tls_ver = ngx_ssl.get_tls1_version_str()
   if request_tls_ver then 
-    request_tls = { version = request_tls_ver } 
+    request_tls = { 
+      version = request_tls_ver, 
+      cipher = var.ssl_cipher,
+      client_verify = var.ssl_client_verify
```

Comment:
Do we really need the `$ssl_client_verify` variable here? Even for mTLS use case this variable is not used as we do validation dynamically using `luaossl` with client supplied certificate. Not sure if `$ssl_client_verify` will contain any useful information in this case.

Reply:
@dndx It seems like `$ssl_client_verify` is [changeable](http://lxr.nginx.org/source/src/http/modules/ngx_http_ssl_module.c#0343). I think it would make sense to update it in our own mTLS logic to reflect client cert verification that occurred in our own logic. This way, we could merge this PR as-is and expect this variable to work today for native NGINX mTLS support, and forward-compatibility later with Kong's mTLS. Any thoughts on that?

Reply:
We will go ahead with the current implementation of this PR, and @dndx will make sure that our mTLS support considers `$ssl_client_verify`. This is good to go now.